README.md

```diff
@@ -48,6 +48,7 @@ The modular IoT Device Client consists of a “base client” and discrete “cl
  Detect feature](https://docs.aws.amazon.com/iot/latest/developerguide/detect-device-side-metrics.html) in the AWS IoT Device Defender service.
 * The client-side [Fleet Provisioning feature](https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html) enables you to replace provisional credentials with device-specific ones
   when you onboard a fleet of devices to AWS IoT Core. It creates a device specific certificate and private key, and registers the device on AWS IoT Core.
+* The client-side [Named Shadows feature](https://docs.aws.amazon.com/iot/latest/developerguide/iot-device-shadows.html) enables you to control your IoT device using AWS IoT Named Shadows. Shadows can store your device's state information and make it available to your device, AWS IoT services, your custom  apps and other AWS services whether the device is online and connected to AWS IoT or not.  
```

Comment:
Please remove the hyperlink from [Named Shadows feature]  and attach it to [AWS IoT Named Shadows]. 
That hyperlink redirects to the cloud side developer guide.